ledger/tests/shred.rs

```diff
@@ -233,7 +232,7 @@ fn setup_different_sized_fec_blocks(
             chained_merkle_root,
             next_shred_index,
             next_code_index,
-            true, // merkle_variant
+            false,
```

Comment:
I think this one should have been updated to use `entries_to_merkle_shreds_for_tests()` as well ?

Reply:
yeah this was one of the tests that was fixed in master.

Reply:
Either way it is correct now=)